### DISCUSSION TITLE
Canalyse Hardware

### DISCUSSION THREAD

**Discussion by damienmaguire** [collin80/SavvyCAN#365] [Ideas]
Wondering if SavvyCAN could be compatible with the Canalyze hardware :
https://kkuchera.github.io/canalyze/

**Comment by collin80** [comment]
That should work fine. That unit apparently is natively supported for socketcan so under linux it should work like any other socketcan device. It doesn't appear to have any prevision to work with Windows or MacOS so probably wouldn't be so useful for either of those.

**Comment by mickeyl** [comment]
I can confirm that it works fine. I'm not too happy with the default firmware though, luckily there is a replacement @ https://github.com/candle-usb/candleLight_fw.
